swagger.json

```diff
@@ -5101,15 +5101,41 @@
                 "label",
                 "value"
             ],
+            "description": "A field of a document. A field can be a string type, number type, or photo type. Between stringValue, numberValue, and photoValue, only one can exist for a single document field depending on the field's type.",
             "properties": {
                 "label": {
                     "type": "string",
                     "description": "Descriptive name of this field.",
                     "example": "Fuel Cost ($)"
                 },
                 "value": {
-                    "description":  "Value of this field. Depending on what kind of field it is, this may be one of the following: an array of image urls, a float, an integer, or a string.",
+                    "description":  "DEPRECATED: Please use stringValue, numberValue, or photoValue instead. Value of this field. Depending on what kind of field it is, this may be one of the following: an array of image urls, a float, an integer, or a string.",
```

Comment:
It's fine to add `DEPRECATED` to the description, but please also add the `"deprecated": true` OpenAPI 3.0 annotation to this value.

Reply:
Ooh, great I didn't know this existed!

Reply:
When I add `"deprecated": true` under `"value"`, it says "should not have additional properties additionalProperties: deprecated" on editor.swagger.io. Do you think the editor is just wrong?